### ISSUE TITLE
Remove outdated "Other Styles" entry

### ISSUE THREAD

**Issue by zen-crab** [airbnb/javascript#2492]
In the Resources -> 'Other Styles' section of the README, there is a link to 'Popular JavaScript Coding Conventions on GitHub'. That site shows the "current" JavaScript coding conventions by analyzing commits, however it has not been updated since July 19, 2014.

**Comment by ljharb** [comment]
It's still a great report, and highly likely to be much the same. I'd hope to see it updated (cc @outsideris)

**Comment by outsideris** [comment]
Thank you for remembering me. :) I will consider if I can make time to restart the proejct.

**Comment by notkonkona** [comment]
Hey I am a newcomer in this project and I would like to contribute (non-code contributions).

**Comment by ljharb** [comment]
@notkonkona feel free to help with anything labeled “pull request wanted”.

**Comment by ljharb** [comment]
@outsideris i'm still interested in this fwiw - if it's something that just requires paying for infra, i think i can make it happen.

**Comment by ayesha-fir** [comment]
Hello, Myself Ayesha Firaus!
I wanted to contribute to learn some new skills.
Can you please assign me some tasks and give me an opportunity to become a contributor?

Thank you,
Waiting for a positive response :)

**Comment by frazie** [comment]
hello, I'd like to help out with this task if it's still vital to the project
Thanks

**Comment by ljharb** [comment]
@frazie i don't think anyone can help with this but @outsideris.

**Comment by MintuKumar07** [comment]
Hey I am a newcomer in this project and I would like to contribute

**Comment by SuperexMack** [comment]
Yeah i just checked that readme file and i found that yes the "Popular JavaScript Coding Conventions on GitHub" is not updated so now what to do 
1> Remove that part?
 or
2> Interchange that link with other link?
![Uploading Screenshot 2024-05-21 225828.png…]()

**Comment by ljharb** [comment]
Please leave this issue open.

**Comment by ljharb** [comment]
Github's "all issues" views are useless because they don't allow you to remove things without closing them, which isn't always appropriate. I appreciate your frustration, but it's better to just not use that view than to try to artificially empty it.

**Comment by ljharb** [comment]
It's still planned.
